frontend/src/views/CarOwnerUserProfile.js

```diff
@@ -1,31 +1,158 @@
-//Photo
-// welcome name
-//General info: name surname username, email, change password, insert your car for sale
-//, regsitered cars, cars for sale, 
-
-import React, { Component } from "react";
-import {
-    fade,
-    ThemeProvider,
-    withStyles,
-    makeStyles,
-    createMuiTheme,
-  } from '@material-ui/core/styles';
+import React, { Component, useState } from "react";
+import { makeStyles } from '@material-ui/core/styles';
 import FormControl from '@material-ui/core/FormControl';
 import InputLabel from '@material-ui/core/InputLabel';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Button from '@material-ui/core/Button';
+import styles from "./UserProfile.css";
+import mechanic from "../assets/mechanic1.png";
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import ListGroup from "react-bootstrap/ListGroup";
+import ListGroupItem from "react-bootstrap/esm/ListGroupItem";
```

Comment:
We are using material-ui for all of our UI related design, Switching to **react-bootstrap** just for One List Group is by many means not a good choice. The same can be achieved through [Material-UI Grids](https://material-ui.com/components/grid/#grid)

However, since the "damage" is already done, the best course of action here is to merge and refactor in another PR